wiki/Beatmap_Discussion/en.md

```diff
@@ -134,30 +140,49 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 - Mark as Resolved: The creator may select this when the issue has been addressed and either accepted or denied.
 - Reopen a Discussion: If the submission needs further discussion, the creator or mod submitter may reopen the submission by responding and unchecking Resolved.
 
-## Earning Kudosu!
+## Hype, Nomination, and Kudosu!
+
+### Hype
 
-*Note: You get ten votes every hour.*
+Hype is used to help push beatmaps forward to nomination; similar to how Star Priority was previously used. Once a beatmapset has received 5 Hype, it becomes eligible for Nomination. Please note the following:
+
+- Each user has a limited supply of hype (5 per month to start).
+  - You will regain a hype 30 days after that one hype was used.
+- Hype can only be used after playing through a difficulty from a beatmapset.
+
+### Nomination
+
+Nomination is similar to the previous modding system. A beatmap must receive 2 nominations from two [Beatmap Nominators](/wiki/Beatmap_Nominators) before being automatically moved to the qualification stage. Beatmaps must have all Suggestions and Problems resolved before it can be ranked. BN/QAT now can reset nominations by posting a problem in the discussion field.
+
+### Kudosu!
 
 ![](img/thumbs.jpg "Thumbs up for Kudosu!")
 
-Modders can now earn Kudosu! by having a thumbs up on their mod. You can (and probably should) go through old mods on your beatmaps and give them a thumbs up to support the modders before this feature was implemented.
+Giving a thumbs up to a suggestion or problem in the discussion field will earn the modder Kudosu!. The amount of thumbs up received determines how much Kudosu! the modder earns. Any user besides the poster can thumbs up any post. *Note: You get 60 votes every hour.*
```

Comment:
will earn -> will give
Giving a thumb up
earns -> gets
can thumb up

Reply:
> Giving a thumb up

could you explain why it should be `Giving a thumb up` instead of `Giving a thumbs up`? (I messaged this in the \#osu-wiki saying that using "thumbs up" just flows better when reading it)

---

> can thumb up

ditto

Reply:
Thumb up is singular I think